backend/deepchecks_monitoring/logic/data_ingestion.py

```diff
@@ -198,29 +199,44 @@ async def log_labels(
     }
 
     validator = t.cast(t.Callable[..., t.Any], fastjsonschema.compile(labels_table_json_schema))
-
+    errors = []
     for sample in data:
         try:
             validator(sample)
-        except fastjsonschema.JsonSchemaValueException:
-            pass
-            # TODO: new table for model ingestion errors?
+        except fastjsonschema.JsonSchemaValueException as e:
+            errors.append({
+                "sample": str(sample),
+                "sample_id": sample.get(SAMPLE_ID_COL),
+                "error": f"Exception saving label: {str(e)}, for id: {sample.get(SAMPLE_ID_COL)}",
+                "model_id": model.id,
+            })
         else:
-            error = None
             # If got same index more than once, log it as error
-            if sample[SAMPLE_ID_COL] in valid_data:
-                error = f"Got duplicate sample id: {sample[SAMPLE_ID_COL]}"
+            if sample[SAMPLE_ID_COL] in logged_ids:
+                errors.append({
+                    "sample": str(sample),
+                    "sample_id": sample.get(SAMPLE_ID_COL),
+                    "error": f"Got duplicate label for sample id: {sample[SAMPLE_ID_COL]}",
```

Comment:
maybe indicate what was the label you took and what was the label you discard...